src/main/kotlin/com/hedvig/accountservice/schedulers/domain/commands/ApproveChargeCommand.kt

```diff
@@ -1,11 +1,13 @@
 package com.hedvig.accountservice.schedulers.domain.commands
 
 import org.axonframework.commandhandling.TargetAggregateIdentifier
+import java.time.LocalDate
 import javax.money.MonetaryAmount
 
 data class ApproveChargeCommand(
     @TargetAggregateIdentifier
     val memberId: String,
     val amount: MonetaryAmount,
-    val approvedBy: String
+    val approvedBy: String,
+    val dateApproved: LocalDate
```

Comment:
Just how I usually see variable names 🤷 
```suggestion
    val approveDate: LocalDate
```